### PR TITLE
Update save-classic.md

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812. Approved.

File: docs/search/search-query-language/search-operators/save-classic.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ save [append]\<myFolder/mySubFolder/myFileNam\>
 
 ## Rules
 
-* The file size limit for saved data is 500MB.
+* You can successfully query a lookup as long as the file size is less than 8MB, and you can save up to 500MB of data. You will get **File Too Big** error, if the file size exceeds 8MB.
```
Comment (JainM6) [inline docs/search/search-query-language/search-operators/save-classic.md:21]:
```suggestion
* You can successfully query a lookup as long as the file size is less than 8MB, and you can save up to 500MB of data. If the file size exceeds 8MB, you will get the **File Too Big** error.
```

Comment (JainM6) [reply docs/search/search-query-language/search-operators/save-classic.md:21]: Emphasize the condition that triggers the error message.

**Review by JainM6 — DISMISSED** [review]
Minor change

**Pull request merged**